core/src/main/java/com/linecorp/armeria/internal/server/RouteDecoratingService.java

```diff
@@ -162,7 +163,7 @@ public String toString() {
                           .toString();
     }
 
-    private static class InitialDispatcherService extends SimpleDecoratingHttpService {
+    public static final class InitialDispatcherService extends SimpleDecoratingHttpService {
```

Comment:
I kept `InitialDispatcherService` as the nested class because `InitialDispatcherService` and `RouteDecoratingService` are tightly coupled.